internal/cli/mongoshellOpts.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2021 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package cli
+
+import "github.com/mongodb/mongocli/internal/config"
+
+type MongoShellOpts struct{}
+
+// ConfigMongoShellPath returns the MongoDB Shell path.
+func (opts *MongoShellOpts) ConfigMongoShellPath() string {
+	return config.MongoShellPath()
+}
```

Comment:
FYI given how simple this struct is (ie not properties, you could've just use `config.MongoShellPath()` where you needed it, feel free to keep the struct as it could make injecting a config easier but flagging this could be removed if you want to